test/integration/istio/metrics_otlp_input_test.go

```diff
@@ -70,8 +72,8 @@ var _ = Describe(suite.ID(), Label(suite.LabelIntegration), Ordered, func() {
 		// Create 2 deployments (with and without side-car) which would push the metrics to the metrics gateway.
 		podSpec := telemetrygen.PodSpec(telemetrygen.SignalTypeMetrics)
 		objs = append(objs,
-			kitk8s.NewDeployment(metricProducer1Name, backendNs).WithPodSpec(podSpec).K8sObject(),
-			kitk8s.NewDeployment(metricProducer2Name, istiofiedBackendNs).WithPodSpec(podSpec).K8sObject(),
+			kitk8s.NewDeployment(metricProducer1Name, backendNs).WithPodSpec(podSpec).WithPersistentLabel().K8sObject(),
+			kitk8s.NewDeployment(metricProducer2Name, istiofiedBackendNs).WithPodSpec(podSpec).WithPersistentLabel().K8sObject(),
```

Comment:
PersistentLabel is only needed for upgrade tests, so no need to add it here

Reply:
fixed